src/entity/Human.php

```diff
@@ -190,11 +190,10 @@ public function getHungerManager() : HungerManager{
 		return $this->hungerManager;
 	}
 
-	public function consumeObject(Consumable $consumable) : bool{
-		if($consumable instanceof FoodSource && $consumable->requiresHunger() && !$this->hungerManager->isHungry()){
+	public function consumeObject(Consumable $consumable, bool $forceEating = false) : bool{
+		if (($consumable instanceof FoodSource) && !$forceEating && $consumable->requiresHunger() && !$this->getHungerManager()->isHungry()){
```

Comment:
Perhaps you should put it on several lines to make the condition more legible.

Reply:
Plus prefer conditions that consume less first like the variable check before the instance.

Reply:
> Plus prefer conditions that consume less first like the variable check before the instance.

Can you explain it further?